bin/pycbc_live

```diff
@@ -110,7 +111,11 @@ class LiveEventManager(object):
         if self.run_snr_optimization:
             # preestimate the number of CPU cores that we can afford giving
             # to followup processes without slowing down the main search
-            available_cores = len(os.sched_getaffinity(0))
+                # if the machine isn't a mac
+            if platform.system() != 'Darwin':
+                available_cores = len(os.sched_getaffinity(0))
+            else:
+                available_cores = multiprocessing.cpu_count()
```

Comment:
For readability, could you wrap this code into a self-contained function `get_cpu_core_count()` that you place somewhere else in PyCBC, for example in the toplevel `__init__.py` file? I would like to remove code from `pycbc_live` rather than adding it.

Reply:
Also, I think the two methods answer different questions, but in practice this is only a concern when running a "serious" analysis. One would not worry about this while testing. So in some sense, just returning "1" if `os.sched_getaffinity()` is not available would probably be an equally good solution.

Reply:
@titodalcanton I refactored this so that we don't need the multiprocessing.cpu_count fallback on a mac.

I did not move this to `__init__.py` as `os.sched_getaffinity` is probably not the generic function that we might want to use in a generic `get_cpu_core_count` call ... and we would want such a call to work, and not just return 1, on a mac (e.g. I might want to run a reasonable size inference on my mac, using many of the cores). In return, I'll promise to put in some work over the Summer refactoring pycbc_live to move code out of the main executable.